build-release-tools/lib/Builder.py

```diff
@@ -163,8 +163,25 @@ def sudo_creds(self):
     @sudo_creds.setter
     def sudo_creds(self, sudo_creds):
         self._sudo_creds = sudo_creds
-        
-    def to_string(self):
+    
+    @staticmethod
+    def import_environment_file(env_file):
```

Comment:
Great fix, but it will be appreciated if the comments can written in code, to tell other people about the purpose of this function( to address this kind of tricky issue)